ibl_to_nwb/updated_conversion/datainterfaces/iblstreaminginterface.py

```diff
@@ -24,6 +25,46 @@ def __init__(self, **kwargs):
         self.stream_name = kwargs["stream_name"]
         super().__init__(**kwargs)
 
+        one = ONE(base_url="https://openalyx.internationalbrainlab.org", password="international", silent=True)
+        atlas = AllenAtlas()
+        brain_regions = BrainRegions()
+
+        spike_sorting_loader = SpikeSortingLoader(
+            eid=self.session, one=one, pname=self.stream_name.split(".")[0], atlas=atlas
+        )
+        _, _, channels = spike_sorting_loader.load_spike_sorting()
+
+        self.has_histology = False
+        if spike_sorting_loader.histology not in ["alf", "resolved"]:
+            return
+        self.has_histology = True
+
+        ibl_coords = np.empty(shape=(384, 3))
+        ibl_coords[:, 0] = channels["x"]
+        ibl_coords[:, 1] = channels["y"]
+        ibl_coords[:, 2] = channels["z"]
+
+        try:
+            ccf_coords = atlas.xyz2ccf(ibl_coords)  # Sometimes this can fail to map and raises an error
```

Comment:
Do you know what causes the error here? Can you catch the error specifically?

Reply:
Yeah, their IBL Coordinate system is sometimes 'outside' of Allen CCF so it doesn't know how to map them. Only happens in ~2/350 sessions

Enhanced error catching to be more specific